packages/venia-concept/src/components/ProductFullDetail/__tests__/ProductFullDetail.spec.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
I think we're trying to move away from `enzyme` in our tests in favor of `react-test-renderer`.

Reply:
Enzyme still seems to be better at shallow rendering, so there may be an argument for keeping it strictly for non-DOM tests, but I've made this change.